recipes/pytest-mypy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pytest-mypy" %}
+{% set version = "0.4.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5a5338cecff17f005b181546a13e282761754b481225df37f33d37f86ac5b304
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - mypy >=0.500
+    - pip
+    - pytest >=2.8
+    - filelock
+    - python >=3.4
+    - setuptools
+  run:
+    - mypy >=0.500
+    - pytest >=2.8
+    - filelock
+    - python >=3.4
+    - setuptools
```

Comment:
Do you need `setuptools` for the program to run?

Reply:
same as above, this is not needed to run. I just had it from the conda skeleton command result. I will also remove